portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/sites/friendlyurl/FriendlyURL.testcase

```diff
@@ -26,53 +26,28 @@ definition {
 		}
 	}
 
-	@description = "This is a use case for LPS-161024."
+	@description = "This is a use case for LPS-161024. Verify uppercase virtual host names added to a site's url are converted to lowercase."
 	@priority = "3"
-	test CanUseCapitalizedVirtualHostName {
-		property custom.properties = "virtual.hosts.valid.hosts=localhost,127.0.0.1,liferay-lower:virtual.hosts.valid.hosts=localhost,127.0.0.1,LIFERAY-UPPER";
-		property test.name.skip.portal.instance = "FriendlyURL#CanUseCapitalizedVirtualHostName";
+	test CannotUseCapitalizedVirtualHostName {
```

Comment:
Try to find a better name for this, since it implies that you cannot create a virtual instance with a capitalized name, it is tecnically true, it just gets converted to lowercase. We want to avoid a misinterpretation for anyone else who looks at this test if we can